include/circt/Dialect/FIRRTL/FIRRTLOpInterfaces.h

```diff
@@ -21,11 +21,11 @@ namespace firrtl {
 
 class FIRRTLType;
 
-enum class Direction { Input = 0, Output };
+enum class Direction { In, Out };
 
 template <typename T>
 T &operator<<(T &os, const Direction &dir) {
-  return os << (dir == Direction::Input ? "input" : "output");
+  return os << (dir == Direction::In ? "input" : "output");
```

Comment:
Would it make sense to rename the printed strings as well?  I don't think that "put" is doing much for us.

Reply:
Yeah, I think this is unused as is.  I'll pull this change out of the other PR as well and use it when printing port directions for FModuleOp.